src/main/kotlin/com/github/jensim/megamanipulator/settings/MySettings.kt

```diff
@@ -228,5 +228,10 @@ sealed class CodeHostSettings
         init {
             validate()
         }
+
+        companion object {
+            const val GITHUB_USERNAME = "GITHUB_USERNAME"
```

Comment:
These consts dont belong in main code?

Reply:
It's in the MySettings

Reply:
What I mean is, the prop should never be used in the "prod" code

Reply:
I got your point, it makes sense